src/components/Nav/Nav.js

```diff
@@ -1,6 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Nav.scss';
 
-const Nav = () => <h1>Nav</h1>;
+const Nav = () => {
+  return (
+    <header className="header">
```

Comment:
최상위 태그의 className은 해당 컴포넌트명을 camelCase로 작성해주세요!
그리고 nav이기 때문에 `<nav>` 태그 사용하시면 될 것 같습니다.
```suggestion
    <header className="nav">
```

Reply:
확인했습니다! 리뷰 반영하겠습니다 👍